README.md

```diff
@@ -30,6 +30,11 @@ To install knoxite, simply run:
     cd knoxite
     go build ./cmd/knoxite/
 
+Or use your favourite package manager:
+
+    # Arch Linux (btw)
+    yay -S knoxite
```

Comment:
Currently that would still be `knoxite-git`.

Reply:
Thanks for that, I simply assumed it's available as `knoxite` in the AUR